src/pkg/cli/client/byoc/baseclient.go

```diff
@@ -100,14 +100,14 @@ func (b *ByocBaseClient) RemoteProjectName(ctx context.Context) (string, error)
 	// Get the list of projects from remote
 	projectNames, err := b.projectBackend.BootstrapList(ctx)
 	if err != nil {
-		return "", err
+		return "", fmt.Errorf("no cloud projects found: %w", err)
```

Comment:
I think this is a step in the right direction, but it still isn't clear what the user should do. Maybe we can make a suggestion?

Reply:
I get your point, but this function doesn't know what the user is trying to do.